Index: configure
===================================================================
--- configure
+++ configure	2018-09-13 03:45:41.468832908 +0200
@@ -16192,7 +16192,7 @@
 # Package dependencies
 GLIB_REQUIRED=2.55.0
 LIBWNCK_REQUIRED=2.91.0
-LIBGTOP_REQUIRED=2.37.2
+LIBGTOP_REQUIRED=2.32.0
 GTK_REQUIRED=3.22.0
 GNOME_ICON_THEME_REQUIRED=2.31
 GTKMM_REQUIRED=3.3.18
Index: configure.ac
===================================================================
--- configure.ac
+++ configure.ac	2018-09-13 03:05:48.652890230 +0200
@@ -29,7 +29,7 @@
 # Package dependencies
 GLIB_REQUIRED=2.55.0
 LIBWNCK_REQUIRED=2.91.0
-LIBGTOP_REQUIRED=2.37.2
+LIBGTOP_REQUIRED=2.32.0
 GTK_REQUIRED=3.22.0
 GNOME_ICON_THEME_REQUIRED=2.31
 GTKMM_REQUIRED=3.3.18
